languages/python/exercises/concept/tuples/.docs/after.md

```diff
@@ -0,0 +1,145 @@
+In Python, a tuple is an immutable collection of items in _sequence_. Like most collections, tuples can hold any (or multiple) data type(s) -- including other tuples. Like any sequence, items are referenced by 0-based index number. Tuples support all [common sequence operations](https://docs.python.org/3/library/stdtypes.html#common-sequence-operations).
+
+Tuples take up very little memory space compared to other collection types and have constant (_O(1)_) access time. However, they cannot be resized, sorted, or altered once created, so are less flexible when frequent changes or updates to data are needed.
```

Comment:
This is precisely the sort of information to put in an `after.md` file. 👍

Reply:
Maybe also make `../../../../reference/concepts/builtin_types/tuple.md` an absolute link?